.gitignore

```diff
@@ -25,6 +25,7 @@
 **/.DS_Store
 
 # Maven artifacts (i.e. object files)
+**/pom.xml.versionsBackup
```

Comment:
This is present because the `mvn versions:set` command creates backups of the pom.xml files it edits, but we don't want to commit those.